src/main/java/io/vertx/ext/stomp/StompServer.java

```diff
@@ -0,0 +1,163 @@
+   * @return the created {@link StompServer}
+   */
+  static StompServer create(Vertx vertx) {
+    return create(vertx, new StompServerOptions());
+  }
+
+  /**
+   * Configures the {@link StompServerHandler}. You must calls this method before calling the {@link #listen()} method.
+   *
+   * @param handler the handler
+   * @return the current {@link StompServer}
+   */
+  @Fluent
+  StompServer handler(StompServerHandler handler);
+
+  /**
+   * Connects the STOMP server to the given port.
+   *
+   * @param port the port
+   * @return the current {@link StompServer}
+   */
+  @Fluent
+  StompServer listen(int port);
+
+  /**
+   * Connects the STOMP server to the given port / interface.
+   *
+   * @param port the port
+   * @param host the interface
+   * @return the current {@link StompServer}
+   */
+  @Fluent
+  StompServer listen(int port, String host);
+
+  /**
+   * Connects the STOMP server to the port / host configured in the server options.
+   *
+   * @return the current {@link StompServer}
+   */
+  @Fluent
+  StompServer listen();
+
+
+  /**
+   * Connects the STOMP server default port (61613) and network interface ({@code 0.0.0.0}). Once the socket
+   * it bounds calls the given handler with the result. The result may be a failure if the socket is already used.
+   *
+   * @param handler the handler to call with the result
+   * @return the current {@link StompServer}
+   */
+  @Fluent
+  StompServer listen(Handler<AsyncResult<StompServer>> handler);
+
+  /**
+   * Connects the STOMP server to the given port. This method use the default host ({@code 0.0.0.0}). Once the socket
+   * it bounds calls the given handler with the result. The result may be a failure if the socket is already used.
+   *
+   * @param port    the port
+   * @param handler the handler to call with the result
+   * @return the current {@link StompServer}
+   */
+  @Fluent
+  StompServer listen(int port, Handler<AsyncResult<StompServer>> handler);
+
+  /**
+   * Connects the STOMP server to the given port / interface. Once the socket it bounds calls the given handler with
+   * the result. The result may be a failure if the socket is already used.
+   *
+   * @param port    the port
+   * @param host    the host / interface
+   * @param handler the handler to call with the result
+   * @return the current {@link StompServer}
+   */
+  @Fluent
+  StompServer listen(int port, String host, Handler<AsyncResult<StompServer>> handler);
+
+  /**
+   * Closes the server.
+   *
+   * @param completionHandler handler called once the server has been stopped
+   */
+  void close(Handler<AsyncResult<Void>> completionHandler);
+
+  /**
+   * Closes the server.
+   */
+  void close();
+
+  /**
+   * Checks whether or not the server is listening.
+   *
+   * @return {@code true} if the server is listening, {@code false} otherwise
+   */
+  boolean isListening();
+
+  /**
+   * Gets the port on which the server is listening. 0 is not listening.
```

Comment:
0 as port value is usually used to signify "use any port" so I think -1 might be a better choice for "not listening".

Also, when 0 is specified in NetServer we use actualPort() to let the user get the actual port the server is listening on. Might be useful here.